packages/amplify-category-api/src/graphql-transformer/transformer-factory.ts

```diff
@@ -1,36 +1,16 @@
-import { AuthTransformer as AuthTransformerV2 } from '@aws-amplify/graphql-auth-transformer';
```

Comment:
I'm glad we refactored this one. One comment here is that under the `graphql-transformer`, we still have a mix of v1 and v2 files. If we could move all the v2 logic into `amplify-graphql-transform`, that would be awesome.

Reply:
That's a good callout, I'll separate in an upcoming PR.